src/main/webapp/resources/js/pages/projects/settings/components/ReferenceFiles.jsx

```diff
@@ -1,21 +1,21 @@
-import React from "react";
 import { notification, Space, Table, Typography } from "antd";
-import { InfoAlert } from "../../../components/alerts";
-import { getProjectInfo } from "../../../apis/projects/projects";
-
-import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
-import { ContentLoading } from "../../../components/loader";
-import {
-  DownloadTableItemButton,
-  RemoveTableItemButton,
-} from "../../../components/Buttons";
+import React from "react";
```

Comment:
Throughout this file anywhere we have `project.name` we need to change it to `project.label` as the project object doesn't have a key `name` but has `label`

Reply:
Updated in 61f667f